### ISSUE TITLE
Closing the modal prior to connecting causes an infinite loading state

### ISSUE THREAD

**Issue by cmi23** [Web3Auth/web3auth-wagmi-connector#89]
Steps to reproduce:
- Click `connect`
- Close the web3auth modal immediately, before connecting to any provider
- Notice how `wagmi`'s loading state is still set to `true`

https://codesandbox.io/p/github/sumicet/web3auth-bugs/infinite-loading-state?file=/src/App.tsx:7,1

Expected behavior: If the user closes the modal before connecting to a provider, the loading state should revert to `false`.

```
@web3auth/modal: 6.1.3
@web3auth/web3auth-wagmi-connector: 4.0.1
```

**Comment by eruizgar91** [comment]
Yes, I got the same issue. Not sure if there is anyone taking look of this

**Comment by antoinefarley** [comment]
I have the same issue and created a [discussion](https://web3auth.io/community/t/using-web3auth-wagmi-connector-opening-the-modal-sets-wagmis-state-to-connecting-closing-the-modal-doesnt-reset-state/5346) in Web3Auth's community.

**Comment by guillermoriv** [comment]
Hey @antoinefarley I have the same issue, I saw you posted on the community, could you please share your whole logic?, where are you getting from the **accountResult** and also **LOGIN_MODAL_EVENTS**, that would also help me out in my problem, thanks!
